zyanken.py

```diff
@@ -0,0 +1,19 @@
+print('-じゃんけんゲームが始まるぞ！！-\n')
+print('入力した数だけじゃんけんできるぞ！！')
+print('↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓\n')
+while True:
+    try:
+        n:int = int(input('じゃんけんしたい回数(自然数)を入力してね--->'))
+        if n > 0:
+            break
+        else:
+            raise ValueError
```

Comment:
普通は Exceptionクラスだけでraiseはしないです。
以下のようにエラーの概要を文字列で渡して使います。
```suggestion
            raise ValueError("input value is not valid integer. it must be number that larger than 0")
```

Reply:
承知しました。
ご指摘してくださった２箇所を、変更いたしました